CHANGES.rst

```diff
@@ -1076,6 +1076,14 @@ astropy.nddata
 - Fixed ``Cutout2D`` output WCS NAXIS values to reflect the cutout
   image size. [#7552]
 
+- Fixed an bug when creating the ``WCS`` of a cutout (see ``nddata.Cutout2D``)
+  when input image's ``WCS`` contains ``SIP`` distortion corrections by
+  adjusting the ``crpix`` of the ``astropy.wcs.Sip`` (in addition to
+  adjusting the ``crpix`` of the ``astropy.wcs.WCS`` object). This bug
+  had the potential to produce large errors in ``WCS`` coordinate
+  transformations depending on the position of the cutout relative
+  to the input image's ``crpix``. [#7556, #7550]
```

Comment:
I think the adopted convention is to only include the PR#, no the issue #. @pllim will correct me if I'm wrong.

Reply:
@nden From astropy's contributing guidelines:

> Changelog entry: whether you are fixing a bug or adding new functionality, you should add an entry to the `CHANGES.rst` file that includes the PR number and **if possible the issue number** ...

Reply:
Ah, ok, thanks for digging this up.

Reply:
good point @mcara, I think we meant to remove that for a while but it always slipped through.

Reply:
@bsipocz Sorry, I didn't know that. However, it shouldn't hurt to have issue numbers, just in case. But feel free to make another PR to edit out issue number.

Reply:
No, it's definitely an oversight in the docs, see #7560 for a cleanup. The logic is that only the code change is what relevant for the changelog. 
Issues should be referred in the PRs for those interested in more details.

Reply:
He's just trying to confuse me.